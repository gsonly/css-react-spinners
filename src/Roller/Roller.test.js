import React from 'react'
import renderer from 'react-test-renderer'
import { Roller } from '..'

describe('Roller', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Roller />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
