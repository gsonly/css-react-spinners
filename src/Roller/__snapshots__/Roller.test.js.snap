// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Roller renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="lds-roller"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>,
]
`;
